mandatory/2-function-creation.js

```diff
@@ -16,7 +27,17 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if (num >= 100){
+    return "100%"
```

Comment:
this is correct, however, I would prefer to use the parseFloat that you have in line 36. So instead of returning, set the num to 100 and then remove the else statement so that the num is formatted